### PR TITLE
Add auto-renovate for the requirements.txt

### PR DESCRIPTION
**Pull request by ra-rau**
- Add the auto-renovate for the requirements.txt to fix Sphinx-version issues.
- This is partly osism/issue#155

Signed-off-by: Ramona Rautenberg <rautenberg@osism.tech>

### CODE REVIEW

**Comment by berendt** [comment]
I think it is better to define github>osism/renovate-config#python. This way we can control everything via the osism/renovate-config repository.

**Comment by ra-rau** [comment]
How to handle different locations from the requirements.txt?
- https://github.com/osism/testbed/tree/main/docs
- https://github.com/osism/cfg-cookiecutter
This are example repositories

File: .github/renovate.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "extends": [
     "github>osism/renovate-config"
```
Comment (berendt) [inline .github/renovate.json:3]: Use `` "github>osism/renovate-config:python"``

**Comment by berendt** [comment]
Replaced by b2ff75b0cbd9b862bf10374c7ae5bc1d17f6520a

**Pull request closed without merging**